tests/test.py

```diff
@@ -0,0 +1,82 @@
+import unittest
+from app.dojo import Dojo
+from app.rooms import Room, Office, LivingSpace
+
+class TestCreateRoom(unittest.TestCase):
+    """Test cases for creating room"""
+    def setUp(self):
+        self.dojo = Dojo()
+
+    def test_adds_office_successfully(self):
+        """
+        Tests if room of type office can be added successfully
+        it should the increase the number of offices.
+
+        """
+        current_no_offices = len(self.dojo.offices)
+        office = self.dojo.create_room("office", ["A04"])
+        room = Office('kigali')
+        self.assertIsInstance(room, Room)
+        new_no_offices = len(self.dojo.offices)
+        self.assertEqual(office, "Office a04 added successfully \n")
+        self.assertEqual(new_no_offices-current_no_offices,1)
+
+    def test_adds_living_space_successfully(self):
+        """
+        Tests if room of type livingSpace can be added successfully
+        increases the number of living_spaces.
+
+        """
+        current_no_livingSpaces = len(self.dojo.livingSpaces)
+        living_space =self.dojo.create_room("living", ["Arusha"])
+        room = LivingSpace('java')
+        self.assertIsInstance(room, Room)
+        new_no_livingSpaces = len(self.dojo.livingSpaces)
+        self.assertEqual(living_space, "Living space arusha added successfully\n")
+        self.assertEqual(new_no_livingSpaces-current_no_livingSpaces,1)
+
+    def test_creating_room_that_exists(self):
+        """Should not accept duplicating rooms"""
+        self.dojo.create_room("office", ["Bujumbura"])
+        room =self.dojo.create_room("office", ["Bujumbura"])
+        self.assertEqual(room,"Room with name bujumbura already exists \n")
+
+    def test_create_wrong_room_type(self):
+        """
+        Tests adding wrong room type
+        should not increase the number of rooms
+
+        """
+        current_no_rooms = len(self.dojo.all_rooms)
+        room = self.dojo.create_room("social_hall", ["Stam"])
+        new_no_rooms = len(self.dojo.all_rooms)
+        self.assertEqual(room, "Wrong room type \n")
+        self.assertEqual(current_no_rooms, new_no_rooms)
+
+class TestAddingPersons(unittest.TestCase):
+    """Test cases for Adding Persons"""
+    def setUp(self):
+        self.dojo = Dojo()
+
+    def test_successfully_adds_fellow_to_system(self):
+        """
+        Should add fellow to the system
+        and increase the number of fellows
+
+        """
+        current_no_fellows = len(self.dojo.fellows)
+        self.dojo.add_person('Kipyegon Ken', 'Fellow')
```

Comment:
your `add_person` method doesn't reflect the one you are testing here, refactor your tests if you adjust the functionality of the code.